README.md

```diff
@@ -162,6 +162,13 @@ server-tools: # Here we repeat server-tools, but no problem because it's a
     - html_text
 ```
 
+You can add all modules in a repo by using a `*`:
+
+```yaml
+website:
+    - "*"
```

Comment:
Hmm I feel the right design would be to use it as a single string. Possible options:

```
website: "*"
web: all
server-tools: true
```

From the above, please choose only one (the one you like most, I don' really care 😋). Setting it as an array item as you purpose makes not much sense IMHO.